.changeset/config.json

```
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [],
+  "access": "restricted",
```

```suggestion
  "access": "public",
```

ref: [changelong](https://github.com/atlassian/changesets/blob/13406e99ef0927361b36c345f3fc51acf7885e40/packages/get-workspaces/CHANGELOG.md#patch-changes-4)